.env

```diff
@@ -0,0 +1 @@
+ITEZ_API_URI=http://localhost:8000/api/
```

Comment:
Why is this an addition when it already existed ?

Reply:
- When I pulled latest changes from upstream had a conflict and accepted incoming changes I don't know if it treated it as an addition